craft_parts/plugins/base.py

```diff
@@ -47,6 +50,7 @@ def __init__(
     ) -> None:
         self._options = properties
         self._part_info = part_info
+        self._action_properties: ActionProperties
```

Comment:
does this actually work? Is it to satisfy `mypy`?

Reply:
Yes, it's enforcing typing as a hint to mypy or pyright.